msbuild/Xamarin.MacDev.Tasks/Tasks/UnpackLibraryResources.cs

```diff
@@ -47,8 +48,40 @@ public class UnpackLibraryResources : XamarinTask, ITaskCallback, ICancelableTas
 		[Output]
 		public ITaskItem [] UnpackedResources { get; set; } = Array.Empty<ITaskItem> ();
 
+		[Output]
+		public ITaskItem [] AtlasTextures { get; set; } = Array.Empty<ITaskItem> ();
+
+		[Output]
+		public ITaskItem [] ColladaAssets { get; set; } = Array.Empty<ITaskItem> ();
+
+		[Output]
+		public ITaskItem [] CoreMLModels { get; set; } = Array.Empty<ITaskItem> ();
+
+		[Output]
+		public ITaskItem [] ImageAssets { get; set; } = Array.Empty<ITaskItem> ();
+
+		[Output]
+		public ITaskItem [] InterfaceDefinitions { get; set; } = Array.Empty<ITaskItem> ();
+
+		[Output]
+		public ITaskItem [] PartialAppManifests { get; set; } = Array.Empty<ITaskItem> ();
+
+		[Output]
+		public ITaskItem [] SceneKitAssets { get; set; } = Array.Empty<ITaskItem> ();
+
 		#endregion
 
+		enum ResourceType {
+			AtlasTexture,
+			BundleResource,
+			ColladaAsset,
+			CoreMLModel,
+			ImageAsset,
+			InterfaceDefinition,
+			PartialAppManifest,
+			SceneKitAsset,
+		}
+
 		public override bool Execute ()
 		{
 			if (ShouldExecuteRemotely ()) {
```

Comment:
Can we remove the remote execution code or are we still running this task remotely in certain scenarios?